addins/Cake.Fastlane.yml

```diff
@@ -2,9 +2,9 @@ Name: Cake.Fastlane
 NuGet: Cake.Fastlane
 Assemblies:
 - "/**/Cake.Fastlane.dll"
-Repository: https://github.com/RLittlesII/Cake.Fastlane
+Repository: https://github.com/RLittlesII/Cake.Fastlane/
```

Comment:
@Jericho wondering if this is diff "enough" to warrant an update?

Reply:
We just need to decide what we want in our YAML files and I can make sure to standardize URLs. What do you think our standard should be: with or without slash?